_posts/2020-09-21-permission-editor.md

```diff
@@ -0,0 +1,110 @@
+---
+
+title: "Permission editor in admin ui"
+author: [pusztaienike]
+image: "../img/posts/permission_editor.jpg"
+tags: [sensenet, permission]
+
+---
+
+"Nobody can hurt me without my permission. /Mahatma Gandhi/"
+
+---
+
+In the IT world you often handle secure data, which you don't want to share with the whole universe. But how can you hide those from the outsiders? What if you would like to share those only with your group? And another data with another group? 
+
+Of course there is a simple solution for that: permission system. Almost every software has permissions which handle what u can see & do.
+
+I think most of us use any social network. You may search for a person and you cannot see her/his photos, after you apply a friend request everything change.
+You see more than ever. Or what about your company? If you are not the boss I am sure that your company has datas which you do not have the sufficient permission.
+
+Permissions are everywhere. And of course sensenet has its own permission system as well. On the new admin-ui gui we did not have any surface to handle these permissions. Till now. 🕛 In the last couple of weeks we have tried to design and implement the simplest, easiest to use **permission editor**.
+
+## Customize it as you want 🔧🔨
+
+sensenet has a lot of built-in permission and we also keep some for your special needs:  Custom01, Custom02, etc. You can also group these permissions for the permission editor dialog as you want. The related settings file is available under Setup menu / Permission.settings.
```

Comment:
```suggestion
sensenet has a lot of built-in permissions but there are some reserved one that you are free to use in case of you have special needs ([custom permissions](https://docs.sensenet.com/concepts/user-and-permission-management/04-custom-roles-and-permissions#permissions)). You can also group these permissions for the permission editor dialog as you want. The related settings file is available under Setup menu / Permission.settings.
```